From 71c77c4d2381f2d2b8c60ed0c5d42e7b38b05d8d Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Wed, 8 Sep 2021 02:47:22 -0300
Subject: [PATCH 2/2] Increase mouse wheel sensitivity

---
 src/xf86libinput.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/xf86libinput.c b/src/xf86libinput.c
index f14ae52..438afbb 100644
--- a/src/xf86libinput.c
+++ b/src/xf86libinput.c
@@ -1651,7 +1651,7 @@ get_wheel_120_value(struct xf86libinput *driver_data,
 	}
 
 	angle = libinput_event_pointer_get_scroll_value_v120(event, axis);
-	return s->dist * angle/120;
+	return s->dist * angle/22;
 }
 #endif
 
-- 
2.34.1

